CMakeLists.txt

```diff
@@ -25,8 +25,15 @@ endif ()
 set(CMAKE_C_COMPILER_NAMES clang-7 cc)
 set(CMAKE_CXX_COMPILER_NAMES clang++-7 c++)
 
-project("Open Enclave SDK" LANGUAGES C CXX ${OE_ASM})
+project("Open Enclave SDK" LANGUAGES C CXX ${OE_ASM}
+  HOMEPAGE_URL "https://github.com/openenclave/openenclave")
 set(PROJECT_VERSION ${OE_VERSION})
+set(OE_SCRIPTSDIR "${PROJECT_SOURCE_DIR}/scripts")
+
+find_program(OE_BASH bash)
```

Comment:
Let me do a pass through the repo for all-caps BASH references and think about what to do. Andy prefers to use variables or namespaces for commonly used variables/targets, to prevent collision with builds of consumers of our SDK

Reply:
The variable naming is not the main point I was making though; the `BASH` variable explicitly looks up Git Bash for use with the clangw scripts (see comments on line 137), not the one in Windows. I'm guessing that since everything is compiling and running, `find_program` correctly finds Git Bash as well on Windows. The [CMake rules for it](https://cmake.org/cmake/help/v3.0/command/find_program.html) don't make it obvious that's that case though and, particularly if we're using it for samples, have we checked that it robustly finds the right version of bash on Windows?

If nothing else, lines 136-157 are redundant if we decide that `find_program` is sufficient.

Reply:
Oh I see! Thanks Simon, nice spot. I'll address this soon